deployments/warp_routes/USDC/eclipse/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't really understand what these files are for? They're not warp configs so do they belong here in the warp_routes deployments dir?